docs/react-integration.md

```diff
@@ -52,56 +52,51 @@ setInterval(() => {
 
 **提示:** 你可以在 [在线编译器CodeSandbox](https://codesandbox.io/s/minimal-observer-p9ti4?file=/src/index.tsx)中尝试上面的例子。
 
- `observer` HOC 将自动订阅 React components 中任何 _在渲染期间_ 被使用的  _可被观察的对象_ 。
-因此, 当任何可被观察的对象 _变化_ 发生时候 组件会自动进行重新渲染（re-render）。
-它还会确保组件在 _没有变化_ 发生的时候不会进行重新渲染（re-render）。
-但是, 更改组件的可观察对象的不可读属性, 也不会触发重新渲染（re-render）。
+ `observer` HOC 将自动订阅 React 组件中所有 _在渲染期间_ 被使用的 _可观察的对象_ (observables)。因此, 当与之关联的可观察的对象发生 _变化_ 时，组件会自动进行重新渲染。它还会确保组件在 _没有关联的变化_ 发生时，不进行重新渲染。所以，当组件能够访问的、但实际没有被组件读取的可观察对象发生变化时，组件不会重新渲染。
 
-在实际项目中，这一特性使得MobX应用程序能够很好的进行开箱即用的优化，并且通常不需要任何额外的代码来防止过度渲染。
+在实践中，这一特性使得使用MobX的应用程序能够被很好地开箱即用地优化，并且通常不需要任何额外代码来防止过度渲染。
 
-要想让`observer`生效, 并不需要关心这些对象 _如何传递到_ 组件的（它们只要能传递给组件即可 ·译者注）, 只需要关心他们是否是可读的。
-深层嵌套的可观察对象也没有问题, 复杂的表达式类似 `todos[0].author.displayName` 也是可以使用的。
-与其他必须显式声明或预先计算数据依赖关系的框架（例如 selectors）相比，这种发生的订阅机制就显得更加精确和高效。
+要想让`observer`生效, 并不需要关心这些对象是 _如何触达到_ 组件的（译者注：即无需关心组件如何访问这些可观察对象）, 只需要关心他们是否被读取。读取深层嵌套的可观察对象也没有问题, 复杂的表达式类似 `todos[0].author.displayName` 也是可以使用的。与其他必须显式声明或预先计算数据依赖关系的框架（例如 selectors）相比，这种发生的订阅机制就显得更加精确和高效。
 
 
 ## 本地与外部状态（Local and external state）
 
-在 Mobx 可以非常灵活的组织或管理（state）, 从（技术角度讲）它不关心我们如何读取可观察对象，也不关心他们来自哪里。
-下面的例子将通过不同的设计模式去使用被 `observer`包裹的组件。
+在 Mobx 可以非常灵活的组织或管理状态, 从（技术角度讲）它不关心我们如何读取可观察对象，也不关心他们来自哪里。
+下面的例子将通过不同的设计模式去展示如何在 `observer` 包裹的组件中使用外部或本地状态。
 
 ### `observer` 组件中使用外部状态 （Using external state in `observer` components）
 
 <!--DOCUSAURUS_CODE_TABS-->
 <!--使用 props-->
 
-可被观察对象可以通过组件的props属性传入  (在下面的例子中):
+将可被观察对象作为组件的props传入 (如下所示):
 
 ```javascript
 import { observer } from "mobx-react-lite"
 
-const myTimer = new Timer() // See the Timer definition above.
+const myTimer = new Timer() // 请参考之前Timer的定义.
 
 const TimerView = observer(({ timer }) => <span>Seconds passed: {timer.secondsPassed}</span>)
 
-// 通过props传递myTimer.
+// 将myTimer作为props传递.
 ReactDOM.render(<TimerView timer={myTimer} />, document.body)
 ```
 
 <!-- 使用全局变量 -->
 
-虽然我们不关心是 _如何_ 引用（reference）的可观察对象,但是我们可以使用 （consume）
-外部作用域（outer scopes directly）的可观察对象  (类似通过 import这样的方法, 等等)：
+因为我们不关心 _如何_ 引用（reference）可观察对象, 所以我们可以使用 （consume）
+外部作用域（outer scopes）中的可观察对象  (类似通过 import这样的方法, 等等)：
 
 ```javascript
-const myTimer = new Timer() //  Timer 定义在上面.
+const myTimer = new Timer() //  请参考之前Timer的定义.
 
-// 没有props, `myTimer` 立刻变成了闭包。
+// 这里不用props, 而是直接使用闭包中的 `myTimer`。
 const TimerView = observer(() => <span>Seconds passed: {myTimer.secondsPassed}</span>)
 
 ReactDOM.render(<TimerView />, document.body)
 ```
 
-直接使用可观察对象效果很好，但是这通常会是通过模块引入，这种写法可能会使单元测试变得复杂。 因此，我们建议使用React Context。
```

Comment:
Using observables directly works very well, but since this typically introduces module state, this pattern might complicate unit testing. Instead, we recommend using React Context instead.